### PR TITLE
Fix the godeps nonsense with snappy and inf

### PR DESCRIPTION
**Pull request by sethvargo**
I don't know if this is due to my lack of understand about godep, a bug in the tool itself, or some combination thereof, but here's what happened.

Over the past few months, the go team has moved snappy from google code to GitHub. Then, they renamed the package from github.com/golang/snappy/snappy to github.com/golang/snappy. The tool gocql depended on the old version of github.com/golang/snappy/snappy, but was updated to the new canonical path in https://github.com/gocql/gocql/commit/b907cb3a0ed93c1a7bd4074b6fc951c3fd2aa529.

The problem is that the vendored gocql still depended on snappy/snappy, but that doesn't exist anymore. But trying to update gocql would complain that snappy/snappy didn't exist. Then, updating snappy/snappy would complain that snappy/snappy didn't exist. Then inf somehow go involved.

Long story short, I ended up having to manually update snappy and save it in the workspace and then gocql and inf would properly update :frowning:

### CODE REVIEW

**Comment by sethvargo** [comment]
:cry:

**Comment by BRMatt** [comment]
sad
